test/testcafe/spec/EnrollAuthenticatorSecurityQuestion_spec.js

```diff
@@ -38,6 +38,9 @@ test.requestHooks(answerRequestLogger, authenticatorRequiredSecurityQuestionMock
   await t.expect(challengeFactorPageObject.isCreateMyOwnSecurityQuestionTextBoxVisible()).notOk();
   // no signout link at enroll page
   await t.expect(await challengeFactorPageObject.signoutLinkExists()).notOk();
+  // assert go back link shows up
+  await t.expect(await challengeFactorPageObject.goBackLinkExists()).ok();
+  await t.expect(challengeFactorPageObject.getGoBackLinkText()).eql('Go back');
```

Comment:
I Guess we can add a test to make sure if clicking this takes you back to select authenticators page. Can even  be a future jira if u are trying to get this in :)

Reply:
Yea, will create a JIRA for this 👍